### PR TITLE
Use the latest stable cimg/ubuntu 18.04 as our base for CI runs

### PR DESCRIPTION
**Pull request by seanhinde**
This PR was sponsored by the ACF

### CODE REVIEW

**Comment by seanhinde** [comment]
Needed as a minimum to get a newer cmake that will guild rocksdb-1.7.0

**Review by velzevur — APPROVED** [review]

**Review by hanssv — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   base:
     docker:
-      - image: cimg/base:2021.02
+      - image: cimg/base:stable-18.04
```
Comment (dincho) [inline .circleci/config.yml:6]: The year/month is intentional as the most deterministic, isn't it possible to use the same type but newer month ?

Comment (dincho) [reply .circleci/config.yml:6]: i.e `2021.11-18.04`

Comment (seanhinde) [reply .circleci/config.yml:6]: Trouble with that is that it's intentionally broken every so often. Surely better to actually keep up with patches as our users do

Comment (dincho) [reply .circleci/config.yml:6]: I don't mind to keep up with patches, but unintentional updates has been problem to only add additional hassle and always happen exactly when you don't have time for it :)

However, you have 2 approvals feel free to merge it as is :)

**Pull request merged**